addons/subscription/data/subscription_demo.xml

```diff
@@ -1,23 +1,21 @@
 <?xml version="1.0" ?>
-<openerp>
-    <data>
+<odoo>
         <record id="base.user_demo" model="res.users">
```

Comment:
Sir, It was not removed but moved from security.xml file to its demo file in this commit:
https://github.com/odoo/odoo/commit/6caca89f8794fb85ba381cc0755e71d63a9fbbc1

And the group "group_tool_user" was renamed to "group_sale_salesman" in commit: https://github.com/odoo/odoo/commit/d510ea50cee6ada029ad899da7e536c0b1a822b2

:smile: